### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by weibel** [Visput/VPAttributedFormat#2]
I ended up here in search of a category that would add an equivalent of stringWithFormat to  NSAttributedString. 

Basically something like
```objc
[NSAttributedString attributedStringWithFormat:format, arg1_NSAttributedString, arg2_NSString, ...];
```

vp_attributedStringWithAttributedFormat takes an NSAttributedString as format and substitutes arguments into the existing attributes. However it fails if an argument itself is an NSAttributedString partly because the result of `- (NSString *)buildSubstring` is an NSString and partly because the substringsContainer.substrings for loop works on NSStrings and the given attributed format.

I also looked at https://github.com/cocodelabs/NSAttributedString-CCLFormat which has this functionality  but their format parser is not as solid as yours.

My feature request is to add e.g.
```objc
+ (instancetype)vp_attributedStringWithFormat:(NSString *)attributedFormat, ...;
```

Or somehow change vp_attributedStringWithAttributedFormat to accept NSAttributedStrings as arguments. If the format in the format string and the argument collides one of them should take precedence. This sounds more complicated.

**Comment by Visput** [comment]
Good idea, I will work on it. Thanks for request.

**Comment by weibel** [comment]
Great. I'm looking forward to this. Also congrats on the SYNTHESIZE_VALUE_WRAPPER_INTERFACE code. It's really neat.

**Comment by Visput** [comment]
Implemented in v1.2.0.

Feature details:  
NSAttributedString objects can be used as attributed format arguments.
If format attributes have conflict with argument attributes then format attributes used with higher priority.
